ios/Classes/BetterPlayer.m

```diff
@@ -783,10 +774,8 @@ - (void)pictureInPictureController:(AVPictureInPictureController *)pictureInPict
 
 - (void)pictureInPictureController:(AVPictureInPictureController *)pictureInPictureController restoreUserInterfaceForPictureInPictureStopWithCompletionHandler:(void (^)(BOOL))completionHandler {
     [self setRestoreUserInterfaceForPIPStopCompletionHandler: true];
-    if (_isDisablePIPManually) {
-        _isDisablePIPManually = false;
-    } else {
-        _isBackToAppButtonPressed = true;
+    if (_eventSink != nil) {
```

Comment:
memo:
- With this implementation, we will handle logic open `from PIP return button` or `manually` completely on flutter side.